samples/daily-prebuilt/permissions-can-admin/src/components/AdminPanel/AdminPanel.js

```diff
@@ -4,24 +4,27 @@ const ParticipantListItem = ({
   p,
   makeAdmin,
   removeFromCall,
-  isOwner,
+  localIsOwner,
+  localIsAdmin,
   count,
 }) => (
   <li>
     <span>
       {`${count}. `}
-      <b>
-        {p.permissions.canAdmin && 'Admin | '}
-        {p.local && '(You) '}
-      </b>
+      {p.permissions.canAdmin && <b>{p.owner ? 'Owner | ' : 'Admin | '}</b>}
+      <b>{p.local && '(You) '}</b>
       {p.user_name}: {p.session_id}
     </span>{' '}
-    {!p.local && !p.permissions.canAdmin && isOwner && (
+    {!p.local && !p.owner && (localIsAdmin || localIsOwner) && (
       <span className='buttons'>
-        <button className='red-button-secondary' onClick={removeFromCall}>
-          Remove from call
-        </button>
-        <button onClick={makeAdmin}>Make admin</button>
+        {(!p.permissions.canAdmin || localIsOwner) && (
```

Comment:
are both of these required? just `localIsOwner` covers it, no?

Reply:
The thinking here is that the owner should be able to remove anyone, and admins should be able to remove other non-admins. If it was checking for the owner only, then admins couldn't eject normal participants